tools/Makefile

```diff
@@ -89,7 +89,7 @@ $(curdir)/genext2fs/compile := $(curdir)/libtool/compile
 $(curdir)/gengetopt/compile := $(curdir)/libtool/compile
 $(curdir)/gmp/compile := $(curdir)/libtool/compile
 $(curdir)/isl/compile := $(curdir)/gmp/compile
-$(curdir)/libdeflate/compile := $(curdir)/cmake/compile
+$(curdir)/libdeflate/compile := $(curdir)/libdeflate/compile
```

Comment:
uhhh what?

Reply:
I have no idea :D but would appreciate some help here. I was looking at the other stuff above. basically, we only depend on gccc ...

Reply:
delete this line. libdeflate is not dependent on anything after this change.